app/ore/project/io/ProjectFiles.scala

```diff
@@ -41,14 +51,15 @@ class ProjectFiles(val env: OreEnv) {
     * @param newName  New project name
     * @return         New path
     */
-  def renameProject(owner: String, oldName: String, newName: String): Try[Unit] = Try {
+  def renameProject(owner: String, oldName: String, newName: String): Try[Unit] = Try { //TODO
     val newProjectDir = getProjectDir(owner, newName)
     move(getProjectDir(owner, oldName), newProjectDir)
-    // Rename plugin files
-    for (channelDir <- newProjectDir.toFile.listFiles()) {
-      if (channelDir.isDirectory) {
-        for (pluginFile <- channelDir.listFiles()) {
-          move(pluginFile.toPath, getProjectDir(owner, newName).resolve(pluginFile.getName))
+
+    // Move all files
+    for (versionDir <- newProjectDir.toFile.listFiles()) {
+      if (versionDir.isDirectory) {
+        for (pluginFile <- versionDir.listFiles()) {
+          move(pluginFile.toPath, getProjectVersionDir(owner, newName, pluginFile.getName).resolve(pluginFile.getName))
```

Comment:
`getProjectVersionDir(owner, newName, pluginFile.getName)` should be `getProjectVersionDir(owner, newName, versionDir)` because you're resolving it to the `pluginFile.getName` afterwards.